tlair/src/test/scala/at/forsyte/apalache/tla/lir/io/TestCounterexampleWriter.scala

```diff
@@ -118,30 +104,18 @@ class TestCounterexampleWriter extends FunSuite {
         |EXTENDS test
         |
         |(* Constant initialization state *)
-        |
-        |ConstInit ==
-        |TRUE
+        |ConstInit == TRUE
         |
         |(* Initial state *)
-        |
-        |State0 ==
-        |/\ x = 0
-        |/\ y = 8
+        |State0 == x = 0 /\ y = 8
         |
         |(* Transition Trans1 to State1 *)
-        |
-        |State1 ==
-        |/\ x = 1
-        |/\ y = 9
+        |State1 == x = 1 /\ y = 9
         |
         |(* Transition Trans2 to State2 *)
-        |
-        |State2 ==
-        |/\ x = 2
-        |/\ y = 10
+        |State2 == x = 2 /\ y = 10
```

Comment:
actually, I hope that the Toolbox parses that! The leading `/\` might be from the TLC counterexample format.

Reply:
`State2 == x = 2 /\ y = 10` is a legal TLA+ formula, so it should be fine. Leading `/\` is just for multiline parsing. If, however, you're referring to the examples in the `TLCCounterexampleWriter`, I checked it before, only in cases with 2 or more variables is there a leading `/\`